deploy/nuget/devel-reduced/prepare_dal_nuget.sh

```diff
@@ -0,0 +1,150 @@
+    dal_version=${major_version}.${minor_version}.${update_version}
+
+    if [ $1 = "lnx" ]; then
+        dl_postfix=.so.${major_binary_version}.${minor_binary_version}
+    elif [ $1 = "mac" ]; then
+        dl_postfix=.${major_binary_version}.${minor_binary_version}.dylib
+    elif [ $1 = "win" ]; then
+        dl_postfix=_dll.${major_binary_version}.lib
+    fi
+
+    pkg_name=inteldal.devel-reduced.${platform}.${dal_version}
+    dal_root_prefix=${pkg_name}/build/native/daal
+    tbb_root_prefix=${pkg_name}/build/native/tbb
+    echo "Versions:"
+    echo "MAJOR: ${major_version}, MINOR: ${minor_version}, UPDATE: ${update_version}, MAJOR BINARY: ${major_binary_version}"
+    echo Creating $pkg_name
+    mkdir -p ${dal_root_prefix}
+    mkdir -p ${tbb_root_prefix}
+    # nuspec generation
+    sed "s/__VERSION__/${dal_version}/; s/__PLATFORM__/${platform}/; s/__TBB_PLATFORM__/${tbb_platform}/" deploy/nuget/devel-reduced/inteldal.devel-reduced.nuspec.tmpl > ${pkg_name}/inteldal.devel-reduced.${platform}.nuspec
+
+    # ###### #
+    # oneDAL #
+    # ###### #
+
+    # common part
+    cp LICENSE ${pkg_name}
+    # cmake
+    mkdir -p ${dal_root_prefix}/lib/cmake/oneDAL
+    if [ $1 = "mac" ]; then
+        # TODO: add cmake configs for Mac
+        cp __release_lnx/daal/latest/lib/cmake/oneDAL/* ${dal_root_prefix}/lib/cmake/oneDAL
+    else
+        cp __release_${rls_postfix}/lib/cmake/oneDAL/* ${dal_root_prefix}/lib/cmake/oneDAL
+    fi
+    # env script
+    cp -r __release_${rls_postfix}/env ${dal_root_prefix}
+    # interfaces
+    cp -r __release_${rls_postfix}/include ${dal_root_prefix}
+    # dynamic libraries
+    mkdir -p ${dal_root_prefix}/lib/intel64
+    if [ $1 = "win" ]; then
+        declare -a needed_libraries=(onedal_core)
+    else
+        declare -a needed_libraries=(onedal_core onedal_thread)
+    fi
+    for needed_library in "${needed_libraries[@]}"; do
+        cp __release_${rls_postfix}/${lib_path}/${dl_prefix}${needed_library}${dl_postfix} ${dal_root_prefix}/lib/intel64
+    done
+
+    # win-x64 special part
+    if [ $1 = "win" ]; then
+        sed "s/__MAJOR_BINARY_VERSION__/${major_binary_version}/" deploy/nuget/devel-reduced/inteldal.devel-reduced.win-x64.targets > ${dal_root_prefix}/inteldal.devel-reduced.win-x64.targets
+        cp deploy/nuget/devel-reduced/inteldal.devel-reduced.win-x64.xml ${dal_root_prefix}
+
+        mkdir -p ${dal_root_prefix}/redist/intel64
+        cp __release_${rls_postfix}/redist/intel64/onedal_core.${major_binary_version}.dll ${dal_root_prefix}/redist/intel64
+        cp __release_${rls_postfix}/redist/intel64/onedal_thread.${major_binary_version}.dll ${dal_root_prefix}/redist/intel64
+
+        cp __release_${rls_postfix}/lib/intel64/onedal_core_dll.${major_binary_version}.lib ${dal_root_prefix}/lib/intel64
+    fi
+
+    echo "oneDAL ${dal_version} is packed"
+
+    # ###### #
+    # oneTBB #
+    # ###### #
+
+    # download package
+    tbb_version=2021.5.0
```

Comment:
Is there a reason it is not 2021.6.0? Just something I noticed, don't know if you care